src/lib/yang/value.lua

```diff
@@ -78,12 +78,12 @@ function types.decimal64.tostring(val)
    return tostring(val)
 end
 
-types.empty = {}
+types.empty = {ctype='bool'}
```

Comment:
NACK here.  No need to ever specify a value for instances of EMPTY.  Binary serialization should have a separate case for empty that doesn't serialize a value.